metainfo-rename.patch

```diff
@@ -0,0 +1,29 @@
+diff --git a/data/eu.lucaweiss.lpa_gtk.metainfo.xml b/data/eu.lucaweiss.lpa-gtk.metainfo.xml
+similarity index 96%
+rename from data/eu.lucaweiss.lpa_gtk.metainfo.xml
+rename to data/eu.lucaweiss.lpa-gtk.metainfo.xml
+index 15cbc53..bb67209 100644
+--- a/data/eu.lucaweiss.lpa_gtk.metainfo.xml
++++ b/data/eu.lucaweiss.lpa-gtk.metainfo.xml
+@@ -4,7 +4,7 @@ SPDX-FileCopyrightText: 2024 Luca Weiss
+ SPDX-License-Identifier: CC-BY-SA-4.0
+ -->
+ <component type="desktop-application">
+-    <id>eu.lucaweiss.lpa_gtk</id>
++    <id>eu.lucaweiss.lpa-gtk</id>
```

Comment:
why using a different id?

Reply:
I've pushed a commit to the source repo now to rename [...]lpa-gtk.desktop to [...]lpa_gtk.desktop due to this in the appstream spec:

> While hyphens are allowed for legacy compatibility, their usage is strongly discouraged to ensure interoperability of the AppStream ID with other tools such as D-Bus (and thereby making the ID more generic and useful).